addon/services/tour.js

```diff
@@ -322,6 +346,9 @@ export default Service.extend(Evented, {
    * @public
    */
   next() {
+    if (this.fastboot) {
```

Comment:
I'm not sure if we need to check for fastboot in every method, but I suppose it does not hurt. It seems somewhat redundant though. Is there a way we could ensure everything essentially noops in fastboot without the check in each method, I wonder? 🤔

Reply:
I thought about that as well, however I did have any really good idea how to achieve the same effect without guarding everywhere. The question is how safe we want ember-shepherd to be in fastboot mode. For instance should we allow something like this in a ember hook which might get executed in fastboot
```
tour.setStep();
tour.start();
```
and then do we allow any other methods in fastboot? like someone wants to show a specific step when the app has loaded, then we would also need to guard methods like `show()`

Or maybe there is a way to just inject a dummy service in fastboot mode, which essentially does nothing (however I do not know how from the top of my head)

Reply:
I like the idea of a dummy service or potentially having a function that would set all the methods to noops, then set them back when not in fastboot.